movo_common/movo_config/movo_config.bash

```diff
@@ -30,9 +30,14 @@ export RUN_FULL_SYSTEM_TELEOP=true
 
 # Joystick configurations for joystick set MOVO_JOY_IS_ATTACHED if the joystick
 # is physically attached to this PC
-export MOVO_JOY_IS_ATTACHED=false
-export MOVO_JOY_DEV=/dev/input/js0
+# If you are using the logitech controller, put MOVO_JOY_IS_LOGITECH to true and MOVO_JOY_IS_XBOX to false
+# If you want to use the XBox 360 wireless controller, put the opposite instead. 
+# You need to do a sync_robot and a reboot of the Movo for it to apply.
 
+export MOVO_JOY_IS_ATTACHED=true
+export MOVO_JOY_DEV=/dev/input/js0
+export MOVO_JOY_IS_XBOX=true
+export MOVO_JOY_IS_LOGITECH=false
```

Comment:
LOGITECH should be true by default I think, and traditionally the MOVO_JOY_IS_ATTACHED variable was false too

Reply:
Done